### ISSUE TITLE
Feature request: vault export

### ISSUE THREAD

**Issue by BrayanDSO** [reuseman/flashcards-obsidian#14]
It would be nice to export your entire vault to Anki with just one command.
Using the command for every note is very tiring.

**Comment by reuseman** [comment]
Yep absolutely. I did not insert this feature at the beginning because it was not stable enough and each new update could lead to an undesired update of the previous cards. 

I am going to implement it in the following days!

**Comment by caHerrera100** [comment]
Thanks for creating this. It saves a lot of time. 

When vault export is created, it would be nice to have the single note exportation remain an option :) It helps when wanting to be more specific when exporting to Anki.

Thanks again for the time and effort put into this.

My best, Cesar

**Comment by faerch90** [comment]
Kindly bumping this issue :) 
Is there any chance of this being implemented?

Thank you for your work on this great plugin.

**Comment by bathrobe** [comment]
Another kind bump from myself. Whole vault sync would be divine. The large Anki plugin that once did whole vault sync appears to be abandonware. You may be our last hope! :)

**Comment by alexlyee** [comment]
Yet another kind bump ! :) can we make this happen?

**Comment by reuseman** [comment]
This feature is complete in Flashcards 2.0.0. **Flashcards: Update Anki from vault** processes the complete vault with one command, while **Update Anki from current note** remains available for focused updates. Release: https://github.com/reuseman/flashcards-obsidian/releases/tag/2.0.0
